### DISCUSSION TITLE
Prisma Migrate non sql

### DISCUSSION THREAD

**Discussion by bryanboateng** [prisma/prisma#13376] [Q&A]
Hello,

I have been developing with Prisma and enjoying it. But I am still reluctant to use it in production because of one issue.
In my project I have data in form of a sqlite database and images that live inside of the file system. With prisma migrate I have the flexibility to change the database using SQL, but what if I want to, for example, add the height and width of an image to the database. How would I access the images? Or what if I want to change the way the images are organised in the file system or rename them? Is there a way to interact with the filesystem, maybe even with typescript for more flexibility?

Thanks in advance :)
Bryan

**Comment by nurul3101** [comment]
Hey @bryanboateng 👋 

When you say images that live in a file system, do you mean you are storing the images on your database server and not on any Blob storage provider like AWS S3?

If I understand correctly you want to move from SQLite to another database? 
And when you say adding width and height to the image do you mean adding columns of width and height in your image table using migrate?
Can you please elaborate on your exact use case?

**Reply by vladi-stevanovic** [reply]
Hello @bryanboateng  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
